sdk/src/index.d.ts

```diff
@@ -17,10 +17,27 @@ import * as BigNum from '@liskhq/bignum';
 import * as cryptography from '@liskhq/lisk-cryptography';
 import * as transactions from '@liskhq/lisk-transactions';
 
-declare const Application: any;
+declare class Application {
+	constructor(genesisBlock: object, config: object);
+	run(): Promise<void>;
+	registerTransaction(transactionClass: object): void;
```

Comment:
is class `object`?

Reply:
True, either should be a function or we can set it to `any`?

Reply:
I think for now we can set it to `any`, but probably it's good to find solution when we go to full typescript

Reply:
Also, second parameter is optional matcher